assets/state-container-toolkit/0500_daemonset.yaml

```diff
@@ -41,7 +41,7 @@ spec:
           seLinuxOptions:
             level: "s0"
         volumeMounts:
-          - name: driver-install-path
+          - name: driver-root
             mountPath: /run/nvidia/driver
```

Comment:
I am using the exact hostPath so that dev-char symlink creation works correctly. I want to make sure the target of the symlink resolves correctly on the host. So for example, we create a symlink `/host-dev-char/195:0` --> `/run/nvidia/driver/dev/nvidia0` in the driver validator container. I guess if we transformed the target path before creating the symlink, then we could always mount the driver install directory at `/driver-root` or `/driver-install-dir`. What do you think?